Java 19 build is grumpy?

```
FAILURE: Build failed with an exception.

* What went wrong:
Could not open settings generic class cache for settings file '/home/runner/work/jspecify/jspecify/settings.gradle' (/home/runner/.gradle/caches/7.5.1/scripts/8fxsx7a5az16qabak9o2v29h1).
> BUG! exception in phase 'semantic analysis' in source unit '_BuildScript_' Unsupported class file major version 63
```

--- a/gradle/wrapper/gradle-wrapper.properties
+++ b/gradle/wrapper/gradle-wrapper.properties
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.5.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.6-bin.zip
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists

